raiden/tests/unit/test_startup.py

```diff
@@ -135,6 +135,22 @@ def test_setup_contracts():
     assert raiden_contracts_in_data(contracts)
     assert service_contracts_in_data(contracts)
 
+    # Goerli production TODO: Uncomment when production contracts are deployed in Goerli
```

Comment:
Can we create a follow up issue for this?

Reply:
done: https://github.com/raiden-network/raiden/issues/3819